src/Symfony/Component/HttpFoundation/ServerBag.php

```diff
@@ -89,6 +89,7 @@ public function getHeaders()
 
         // PHP_AUTH_USER/PHP_AUTH_PW
         if (isset($headers['PHP_AUTH_USER'])) {
```

Comment:
`isset($headers['PHP_AUTH_USER'], $headers['PHP_AUTH_PW'])` sounds reasonable

Reply:
this will break existing behavior.
Currently if second param is missed it replaced by empty string, but both exists.
Buy making 2 exists check we will skip both params further.